manifests/backup/backup-schedule-s3-br.yaml

```diff
@@ -4,6 +4,8 @@ kind: BackupSchedule
 metadata:
   name: demo1-backup-schedule-s3
   namespace: test1
+  # annotations:
+    # iam.amazonaws.com/role: "arn:aws:iam::123456789:role"
```

Comment:
1. Please add kms and sa.
2. The spec also needs to update e.g. `from` is required now, including `backup` and `restore`.
3. Suggest adding examples for aws s3.

Reply:
done